scripts/pi-hole/php/footer.php

```diff
@@ -41,32 +41,21 @@
     </div>
     <!-- /.content-wrapper -->
     <footer class="main-footer">
-<style type="text/css">
-	@-webkit-keyframes Pulse{
-		from {color:#630030;-webkit-text-shadow:0 0 9px #333;}
-		50% {color:#e33100;-webkit-text-shadow:0 0 32px #e33100;}
-		to {color:#630030;-webkit-text-shadow:0 0 9px #333;}
-	}
-	a.lookatme {
-		-webkit-animation-name: Pulse;
-		-webkit-animation-duration: 2s;
-		-webkit-animation-iteration-count: infinite;
-	}
-</style>
-        <div class="pull-right hidden-xs <?php if(isset($core_commit) || isset($web_commit)) { ?>hidden-md<?php } ?>">
+	<!-- Version Infos -->
+        <div class="pull-right hidden-xs <?php if(isset($core_commit) || isset($web_commit)) { ?>hidden-sm<?php } ?>">
```

Comment:
I also thouht about that after I saw your tablet landscape screenshot.
Gladly you can change it to your suggestion, but I would move the space before the by-php-added class into the php code, like this:
`<div class="pull-right hidden-xs hidden-sm<?php if(isset($core_commit) || isset($web_commit)) { ?> hidden-md<?php } ?>">`